tests/main/views/test_services.py

```diff
@@ -244,7 +244,7 @@ def test_disabled_service_has_unavailable_banner(self):
         assert unavailable_banner.exists
         assert unavailable_banner.heading_text() == '{} stopped offering this service on {}'.format(
             self.service['services']['supplierName'],
-            'Tuesday 5 January 2016.',
+            'Thursday 6 January 2000.',
```

Comment:
So this test should still show `Tuesday 5 January 2016`, as the date the supplier made the service unavailable.

Reply:
@katstevens Good spot! Fixed in 47900828620dd5e2f8f03ee21e9364c4711828b1